components/search/MobileSearchBox.tsx

```diff
@@ -31,6 +31,7 @@ function SearchBox({
       <input
         id={"algolia_search"}
         type="search"
+        aria-label="Search for commands"
```

Comment:
just confirming this is all we need to do here? He had feedback around using a "for" attribute and making it clear that enter submits the form. Not sure if the "for" attribute is useful here though:

> There is a search form (I believe) that reads as Ctrl+K. This form should have a valid label saying something like “Search for Commands”. The label should be associated with the input field via “for” attribute. The form should have a “Search Commands” submit button located below the input field as Enter key should not be implied.

Reply:
just did a quick google - misunderstood the ask on first pass. Should be addressed now